src/main/frontend/components/block.cljs

```diff
@@ -2008,7 +2008,7 @@
        [:div.custom-query.mt-2 (get config :attr {})
         (when-not (and built-in? (empty? result))
           (ui/foldable
-           [:div.opacity-70
+           [:div.opacity-70.custom-query-title
```

Comment:
Note to reviewer – This is to enable users to add custom styles to the title section of a custom query via `logseq/custom.css`.